README.md

```diff
@@ -59,7 +59,7 @@ docker run  -d \
 			--volume ~/magic_mirror/config:/opt/magic_mirror/config \
 			--volume ~/magic_mirror/modules:/opt/magic_mirror/modules \
 			--name magic_mirror \
-			MichMich/MagicMirror
+			michmich/magicmirror
```

Comment:
@bastilimbach , yes, it builds successfully. I think dos2unix has nothing to do with it since it fails on _RUN npm install_ command, before first dos2unix call. I've googled this errors message and it shows A LOT issues like this for different containers that uses ARM. Some issues for Mac (https://github.com/docker/docker/issues/23865), some for linux and windows